modules/virt-creating-pvcs-to-store-default-os-images.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-default-os-//images.adoc
+
+[id="virt-creating-pvcs-to-store-default-os-images_{context}"]
+= Creating PersistentVolumeClaims to store default OS images
+
+Follow these steps to create a PersistentVolumeClaim (PVC), which you use to upload and save default OS images.
+
+.Prerequisites
+
+* You must have administrator privileges to upload and save default OS images.
```

Comment:
Should we mention the RBAC role here too? Something like:

You must be logged in as a user with the os-images.kubevirt.io:edit role or an administrator.

Reply:
I think it's great to be that specific. I chnaged it to:

* You must be logged in as a user with the `os-images.kubevirt.io:edit` RBAC role or as an administrator.